salt/modules/zypperpkg.py

```diff
@@ -1328,8 +1331,9 @@ def mod_repo(repo, **kwargs):
     if "priority" in kwargs:
         cmd_opt.append("--priority={}".format(kwargs.get("priority", DEFAULT_PRIORITY)))
 
-    if "humanname" in kwargs:
-        cmd_opt.append("--name='{}'".format(kwargs.get("humanname")))
```

Comment:
You can't just remove an option without deprecation. Please add a deprecation warning under humanname:
```suggestion
    if "humanname" in kwargs:
        salt.utils.versions.warn_until(
            3009,
            "Passing 'humanname' to 'mod_repo' is deprecated, slated "
            "for removal in {version}. Please use 'name' instead."
        )
        cmd_opt.append("--name='{}'".format(kwargs.get("humanname")))
```